.codeclimate.yml

```diff
@@ -36,16 +36,22 @@ plugins:
   phpcodesniffer:
     enabled: true
     config:
-      standard: "PSR1,PSR2"
+      standard: "PSR1,PSR2,phpcs.xml"
 
   sonar-php:
     enabled: true
     checks:
       php:S1192: # Define a constant instead of duplicating this literal "value" X times.
         enabled: false
+      php:S1117: # Rename variable which has the same name as the class property.
+        enabled: false
+      php:S1142:
```

Comment:
What's this one for?

Reply:
Too many return statements. Actually this check is [already done by CodeClimate itself](https://github.com/CuyZ/NotiZ/pull/64/files#diff-0655d430acfa953cae08b6619689d18cR27)

https://github.com/SonarSource/sonar-php/blob/master/php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1142.html